l10n_es_vat_prorate/models/account_move.py

```diff
@@ -24,15 +44,19 @@ class AccountMoveLine(models.Model):
     with_vat_prorate = fields.Boolean(
         string="With Vat prorate",
         help="The line will create a vat prorate",
-        default=lambda self: (
-            self.env.company.with_vat_prorate
-            and (
-                not self.env.company.with_special_vat_prorate
-                or self.env.company.with_special_vat_prorate_default
-            )
-        ),
+        compute="_compute_with_vat_prorate",
+        store=True,
```

Comment:
Cuando se instale este módulo de nuevas con un montón de apuntes existentes, esto va a tardar la vida, así que debéis añadir un pre-init-hook que pre-cree la columna para evitarlo.

Reply:
Hecho! Gracias @pedrobaeza !